plugins/community/scalapb-scala/v0.11.12/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# syntax=docker/dockerfile:1.4
+FROM debian:bullseye-20221114 AS build
+
+ARG TARGETARCH
+
+RUN apt-get update \
+ && apt-get install -y curl
+
+RUN curl -fsSL -o protoc-gen-scala.sh https://repo1.maven.org/maven2/com/thesamet/scalapb/protoc-gen-scala/0.11.12/protoc-gen-scala-0.11.12-unix.sh \
```

Comment:
It might be good to add a comment with a reference to https://scalapb.github.io/docs/scalapbc#using-scalapb-as-a-proper-protoc-plugin - I didn't realize at first that the shell script actually embeds the .class files and is self contained.